lib/client.js

```diff
@@ -75,11 +75,17 @@ function Client (broker, conn) {
   }
 
   this.deliverQoS = function deliverQoS (_packet, cb) {
+    var _authPacket = that.broker.authorizeForward(that, _packet)
```

Comment:
yes, you're right. I didn't realize deliverQoS would deliver everything. I originally thought it only would deliver publishes by default :)